test/tools/ossfuzz/yulFuzzerCommon.cpp

```diff
@@ -24,17 +24,17 @@ void yulFuzzerUtil::interpret(ostream& _os, shared_ptr<yul::Block> _ast)
 {
 	InterpreterState state;
 	state.maxTraceSize = 75;
-	state.maxSteps = 10000;
+	state.maxSteps = 100;
 	Interpreter interpreter(state);
 	try
 	{
 		interpreter(*_ast);
 	}
-	catch (InterpreterTerminated const&)
+	catch (ExplicitlyTerminated const&)
```

Comment:
I think it is a bit confusing to catch `ExplicitlyTerminated` here, but leak the other two exceptions.

Reply:
I agree. I can propagate all catches to the fuzzing harness itself for clarity.